source/content/plugins-known-issues.md

```diff
@@ -1043,6 +1049,10 @@ export ENV=dev
   ln -s ../../files/private/wflogs ./wp-content/wflogs
   ln -s ../files/private/wordfence-waf.php ./wordfence-waf.php
   ln -s ../files/private/.user.ini ./.user.ini
+
+  ![replicate-1](https://github.com/pantheon-systems/documentation/assets/103234849/5b7e8d70-e081-49c8-a71d-09a9f8d66109)
+  ![replicate-2](https://github.com/pantheon-systems/documentation/assets/103234849/72b71341-ed1b-4692-b501-4c4eace7ee41)
+  ![wordfence](https://github.com/pantheon-systems/documentation/assets/103234849/d1ae1e49-025a-4ef6-a8cd-8dbb5fdb0745)
```

Comment:
These 3 images are showing as part of the code block as if they were commands for the customer to type.. I'm pretty sure this is not intended..

![image](https://github.com/pantheon-systems/documentation/assets/87093053/4e1be59f-67fd-4a06-97d9-3cc680a826e2)

Reply:
@whitneymeredith despite your [commit](https://github.com/pantheon-systems/documentation/pull/8550/commits/65295f1603bd29a79bb989b45ca099bb4a38fea9) and the build that I see in the main conversation, I'm still seeing what Brian reported on the preview site.

Reply:
@CdrMarks the code for the images doesn't show on the preview link or on my local. can you tell me where you're seeing them?